### ISSUE TITLE
[cockpit-podman] Basic shell commands like `ls` do not work on the first line of the Console.

### ISSUE THREAD

**Issue by urbenlegend** [cockpit-project/cockpit-podman#1412]
### Explain what happens

I was trying to run a few commands within a podman container and I noticed that the first command always breaks, even on really basic commands. Here's a screenshot of what I mean:
![image](https://github.com/cockpit-project/cockpit/assets/241316/d8522f02-eb8c-41e7-b0a0-c46fa0ad3f1c)

The shell works fine afterwards. It's just the first line that is broken. I've tested this with [Jellyfin](https://hub.docker.com/r/jellyfin/jellyfin) and [Nextcloud](https://github.com/nextcloud/docker/blob/master/.examples/docker-compose/insecure/mariadb/apache/docker-compose.yml)

### Version of Cockpit

cockpit 300.1-1, cockpit-podman 76-1

### Where is the problem in Cockpit?

Navigation & Shell

### Server operating system

Arch Linux

### Server operating system version

up to date as of 9/18/2023

### What browsers are you using?

Firefox

### System log

```shell
No logs appear in journalctl when triggering the bug.
```

**Comment by jelly** [comment]
What jellyfin container do you use? Because the upstream one `docker.io/jellyfin/jellyfin:latest` will have an entrypoint meaning you'll have to either override that or ctrl-c which likely restarts the container.

**Comment by chabad360** [comment]
I've seen this issue as well, with many different containers.

**Comment by jelly** [comment]
Right, but I can't reproduce it. What browser do you use and what containers can you reproduce it with?

**Comment by chabad360** [comment]
Hmm, it happens in Firefox, with any container I've tried iirc. Try postgres 14?

I may look into this at some point this week.

**Comment by jelly** [comment]
Interesting, under Firefox I indeed spotted ^L with our cockpit-website container but that starts 

![image](https://github.com/cockpit-project/cockpit-podman/assets/67428/72069b1b-5ff7-420d-b6e0-5c37fd759cd6)

So if I create a container with that image and then change the Command to `sh` in the create image modal I can reproduce it:

![image](https://github.com/cockpit-project/cockpit-podman/assets/67428/e7d18522-b52b-4bb6-a4e4-dbc623d06e05)

**Comment by urbenlegend** [comment]
> What jellyfin container do you use? Because the upstream one `docker.io/jellyfin/jellyfin:latest` will have an entrypoint meaning you'll have to either override that or ctrl-c which likely restarts the container.
> 
> Container console image for reference. ![image](https://user-images.githubusercontent.com/67428/282013789-232107bb-64ce-4b9e-bef4-9abe5c2f4ddd.png)

Yes I am using the upstream Jellyfin container, but this also happens on the upstream Nextcloud container and the linuxserver SWAG container.

**Comment by jelly** [comment]
I can reproduce it now with the steps above, so when I can find time I'll look into it.

**Comment by cmd430** [comment]
same issue here
workaround i've been using to just press backspace before first command; it clears the bad input and allows the command to run fine
